src/main/java/interface_adapter/break_session/BreakSessionViewModel.java

```diff
@@ -6,7 +6,7 @@
  * View Model of Break Session.
  */
 public class BreakSessionViewModel extends ViewModel<BreakSessionState> {
-
+    public static String ADOPTION_LABEL = "Adopt a new cat!";
```

Comment:
Should be private, and the variable name should also be changed to proper checkstyle format.